ui/js/controllers/perf/graphs.js

```diff
@@ -486,7 +486,17 @@ perf.controller('GraphsCtrl', [
                 });
 
                 //Closes popup on-click of "X"
-                $('#close-popup').bind("click", plotUnselected);
+                $('#close-popup').bind("click", hideTooltip);
+
+                //Checks for clicks outside element data-display and closes popup
```

Comment:
Let's say what this does: "Close pop up when user clicks outside of the graph area"